### PR TITLE
kills some more bugs

### LINKED ISSUE #16053: gangrel giant bat wing slam does not have a cooldown
### Severity

Will kill people IRL if not immediately solved

### Short Description

gangrel giant bat wing slam does not have a cooldown

### Reproduction

1. Obtain giant bat
2. Kindly approach the Security Officer
3. wing slam 200 times
4. Memetext!

### Round ID

Private server

### Test Merges

_No response_

### BYOND Username

_No response_

### Additional information

_No response_

### LINKED ISSUE #19031: HUD bugs
### Severity

Inconvenience

### Short Description

part of HUD stays greyed out, sechud criminal status icons changes are delayed.

### Reproduction

- sechud criminal status icons on people (often) didn't update (only visually) when changed (they stayed the same, or had massive delay at certain times)
- greyed out HUD elements at the top won't become "active" (after slipping, etc.) until you click on them, and you have to start clicking on all of them to restore their opacity.

### Round ID

47377

### BYOND Username

Aquir

### LINKED ISSUE #19061: Demon of Sin abilities not working*
### Severity

Round Breaking

### Short Description

Demon of Sin (Pride, assuming others too) button abilities not working

### Round ID

Round #47416

### Test Merges

#18826

### BYOND Username

Crazy_Drakkon; Player affected: CoIoneISanders

### Was this done on a locally hosted or non-Yogstation hosted server?

No

### Additional information

*Apparently

image

### LINKED ISSUE #19093: Blood Cult speedruns by winning the round instantly upon startup
### Severity

Round Breaking

### Short Description

Blood Cult gamemode can randomly end instantly with a cult win because cult is god.

### Reproduction

1. game rolls blood cult
2. round starts
3. round ends
4. cult win

### Round ID

Round #47464

### Test Merges

#18826 #18498 #18502

### BYOND Username

Cark

### Was this done on a locally hosted or non-Yogstation hosted server?

None

### Additional information

_No response_

### LINKED ISSUE #19095: split personality is fucked
## Test Merges: 
Not Supplied / None
## Reproduction:
description: split personality cant commune while inactive and on top of that if it switches back to the original the split gets shunted out of the body entirely and cant get back in

 Submitted by: zealot (1079798074068709386)

### LINKED ISSUE #19106: Arcane Bolt does not care about spell immunity
### Severity

Round Breaking

### Short Description

When hit by an arcane bolt as chaplain the bolt still did damage even though it showed the fizzle message

### Reproduction

1. Join the round as Chaplain
2. Equip null rod
3. Get shot at with arcane bolt
4. Take damage and have fizzle message

### Round ID

47481

### Test Merges

#18826 #18498 #18502

### BYOND Username

Roromonster

### Was this done on a locally hosted or non-Yogstation hosted server?

None

### Additional information

_No response_

### LINKED ISSUE #19123: Ashwalker Shaman Healing ability doesn't go on cooldown when used, still heals.
## Test Merges: 
Not Supplied / None
## Reproduction:
Description: Ashwalker shaman healing hand is usable on others, still heals, but doesn't go on cooldown, allowing for infinite heailng with no cooldown.

 Submitted by: Tawaporah Tenderpaw (241348083907166219)

### LINKED ISSUE #19125: You dont respawn with robes as a lich.
### Severity

Round Breaking

### Short Description

Normally, when you sacrifice a part of your soul in order to make a phylactery, and are killed, you will respawn at your phylactery's location, fully clad in black wizard robes. I did this one round. Guess what happened? I spawned in buck-naked and thus I ended up losing.

### Reproduction

1. Be Wizard
2. Get 'Bind Soul'.
3. Use on anything.
4. Kill yourself (NOW!!) or die in some other way.
5. Respawn.
6. You are now naked.

### Round ID

47519

### Test Merges

#18826, #18498, #18502

### BYOND Username

Yarinoi

### Was this done on a locally hosted or non-Yogstation hosted server?

No

### Additional information

grrrrr i want my antag token grr grrrr

### LINKED ISSUE #19137: Drakeling only has ollie ability
### Severity

Exploit

### Short Description

Drakeling from dragon egg doesnt have wing flap or fire breath abilities, making it have no practical purpose.

### Reproduction

1. Kill legion
2. Steal dragon egg
3. Hatch it
4. hop on the drakeling 
5. Cant do anything with it other then ollie on tables

### Round ID

47543

### Test Merges

#18826 #18498 #18502

### BYOND Username

UltraNumeron

### Was this done on a locally hosted or non-Yogstation hosted server?

No

### Additional information

_No response_

### LINKED ISSUE #19148: Wizard apprentices get spellbooks of their own.
### Severity

Exploit

### Short Description

Wizard apprentices for some reason now get their own spellbook, allowing for wizard mitosis

### Reproduction

1. be wiz
2. buy apprentice
3. apprentice has spellbook
4. apprentice buy apprentice
5. apprentice apprentice 
6. apprentice apprentice apprentice 
7. apprentice apprentice apprentice apprentice 

### Round ID

Round #47551

### Test Merges

_No response_

### BYOND Username

Cark

### Was this done on a locally hosted or non-Yogstation hosted server?

None

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by tattax**
it was foretold to biome that he should've testmerged my last pr cause i can't test bloodcult alone but he didn't listen....

let's try again

ports: https://github.com/fulpstation/fulpstation/pull/991

closes: https://github.com/yogstation13/Yogstation/issues/19106
closes: https://github.com/yogstation13/Yogstation/issues/19095
hopefully closes: https://github.com/yogstation13/Yogstation/issues/19093
closes: https://github.com/yogstation13/Yogstation/issues/19061
closes: https://github.com/yogstation13/Yogstation/issues/19125
closes: https://github.com/yogstation13/Yogstation/issues/19123
closes: https://github.com/yogstation13/Yogstation/issues/19155
closes: https://github.com/yogstation13/Yogstation/issues/19154
closes: https://github.com/yogstation13/Yogstation/issues/19153
closes: https://github.com/yogstation13/Yogstation/issues/19148
closes: https://github.com/yogstation13/Yogstation/issues/19201
closes: https://github.com/yogstation13/Yogstation/issues/19194
closes: https://github.com/yogstation13/Yogstation/issues/19188
closes: https://github.com/yogstation13/Yogstation/issues/19137
closes: https://github.com/yogstation13/Yogstation/issues/19031
closes: https://github.com/yogstation13/Yogstation/issues/19212
closes: https://github.com/yogstation13/Yogstation/issues/16053
closes: https://github.com/yogstation13/Yogstation/issues/19235
closes: https://github.com/yogstation13/Yogstation/issues/19261
:cl:  
bugfix: fixed some more antag bugs
/:cl:

### CODE REVIEW

**Comment by MajManatee** [comment]
Imagine us testing anything we merge.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/drakeling.dm [hunk 1]
```diff
@@ -147,7 +156,7 @@
 	button_icon = 'icons/obj/wizard.dmi'
 	button_icon_state = "fireball"
 	cooldown_time = 1.6 SECONDS //kinetic gun
-	active_msg = span_notice("You prepare %YOUR fire breath attack")
+	active_msg = span_notice("You prepare the fire breath attack")
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/drakeling.dm:159]: There's another one of this in ``/datum/action/cooldown/spell/pointed/drakeling``

File: yogstation/code/game/gamemodes/battle_royale/battleroyale.dm [hunk 2]
```diff
@@ -225,7 +225,7 @@ GLOBAL_VAR(stormdamage)
 		qdel(I)
 	tfue.equipOutfit(/datum/outfit/battleroyale, visualsOnly = FALSE)
 
-/mob/living/carbon/human/Life()
+/mob/living/carbon/human/Life(seconds_per_tick = SSMOBS_DT, times_fired)
 	. = ..()
 	if(is_battleroyale(src))
 		var/datum/antagonist/battleroyale/gamer = mind.has_antag_datum(/datum/antagonist/battleroyale)
```
Comment (JohnFulpWillard) [inline yogstation/code/game/gamemodes/battle_royale/battleroyale.dm]: This should probably all be moved to ``/mob/living/carbon/human/Life()``, it is easy to break here.

Even better alternative, do what Bloodsuckers do and register signal to comsig_mob_life.

File: code/modules/antagonists/bloodsuckers/powers/_powers.dm [hunk 3]
```diff
@@ -40,6 +40,8 @@
 	var/constant_bloodcost = 0
 	///If the Power has any additional descriptions coming from either 3rd partys or the power itself
 	var/additional_text = ""
+	///Path to the ascended version of the power. For Lasombra bloodsucker powers
+	var/ascended_power = null
```
Comment (JohnFulpWillard) [inline code/modules/antagonists/bloodsuckers/powers/_powers.dm:44]: This should work like Tremere does and be its own subtype I think.
That was one of the main reasons for the datumization of Bloodsucker Clans.

**Review by JohnFulpWillard — CHANGES_REQUESTED** [review]

File: yogstation/code/game/gamemodes/battle_royale/battleroyale.dm (hunk 2 above)
Comment (tattax) [reply yogstation/code/game/gamemodes/battle_royale/battleroyale.dm]: hmm true

File: code/modules/antagonists/bloodsuckers/powers/_powers.dm (hunk 3 above)
Comment (tattax) [reply code/modules/antagonists/bloodsuckers/powers/_powers.dm:44]: if i don't do this soonish i'll do it in a followup pr

**Review by JohnFulpWillard — APPROVED** [review]
We should really port ``update_appearance``

**Comment by cuackles** [comment]
image
 Need to know why these icons are deleted

**Comment by tattax** [comment]
https://github.com/yogstation13/Yogstation/pull/19115/commits/4c06ecf85c8ea67e2a2e028604d1bc0872b83f4f

**Review by cuackles — APPROVED** [review]

File: code/datums/components/curse_of_hunger.dm [hunk 4]
```diff
@@ -57,7 +57,7 @@
 	SIGNAL_HANDLER
 	var/obj/item/at_least_item = parent
 	// Items with no slot flags curse on pickup (because hand slot)
-	if(at_least_item.slot_flags && !(at_least_item.slot_flags & slot))
+	if(at_least_item.slot_flags && !(at_least_item.slot_flags & slotdefine2slotbit(slot)))
```
Comment (JohnFulpWillard) [inline code/datums/components/curse_of_hunger.dm]: Update your PR to master, bitflags for merged :D

File: code/modules/antagonists/space_dragon/space_dragon.dm [hunk 5]
```diff
@@ -26,11 +26,18 @@
...
+		"<b>Through endless time and space we have moved. \
+		We do not remember from where we came, we do not know where we will go. All of space belongs to us.\n\
+		It is an empty void, of which our kind was the apex predator, and there was little to rival our claim to this title.\n\
+		But now, we find intruders spread out amongst our claim, willing to fight our teeth with magics unimaginable, their dens like lights flickering in the depths of space.\n\
+		Today, we will snuff out one of those lights.</b>"
+		)
+	to_chat(owner, span_boldwarning(
+		"You have five minutes to find a safe location to place down the first rift. \
+		If you take longer than five minutes to place a rift, you will be returned from whence you came. \
+		Use ALT-CLICK to wing f	lap around you, throwing anything away but making you tired, requiring careful usage.")
```
Comment (JohnFulpWillard) [inline code/modules/antagonists/space_dragon/space_dragon.dm]:
```suggestion
		Use ALT-CLICK to wing flap around you, throwing anything away but making you tired, requiring careful usage.")
```

Comment (tattax) [reply code/modules/antagonists/space_dragon/space_dragon.dm]: part of the flavor

**Review by JohnFulpWillard — APPROVED** [review]

**Pull request merged**